dhall-json/dhall-to-yaml/Main.hs

```diff
@@ -52,14 +64,19 @@ main :: IO ()
 main = do
     GHC.IO.Encoding.setLocaleEncoding GHC.IO.Encoding.utf8
 
-    options@Options {..} <- Options.execParser parserInfo
+    maybeOptions <- Options.execParser parserInfo
+
+    case maybeOptions of
+        Nothing -> do
+            putStrLn (Data.Version.showVersion Meta.version)
```

Comment:
Having `Nothing` represent the `--version` option seems a bit hacky. Why not use the same schema as in `dhall-to-json`?

Reply:
@sjakobi: It's because the option type for `dhall-to-yaml` it defined in a separate file and it doesn't have a `Version` constructor.  I plan to fix that, but I wanted to keep this change as small as possible